src/util/Constants.js

```diff
@@ -680,7 +680,8 @@ exports.WebhookTypes = [
 ];
 
 function keyMirror(arr) {
-  let tmp = Object.create(null);
-  for (const value of arr) tmp[value] = value;
-  return tmp;
+  return arr.reduce((obj, value) => {
+    obj[value] = value;
+    return obj;
+  }, {});
```

Comment:
It was intentional to use `Object.create(null)`:
![image](https://user-images.githubusercontent.com/24852502/103133940-c0a11200-46ad-11eb-8759-461276f53c51.png)

`Object.create(null)` creates an empty object, without inherited properties nor methods, which is what we use for enums.

```suggestion
  }, Object.create(null));
```